src/app/cluster/details/cluster/edit-cluster/template.html

```diff
@@ -96,7 +96,8 @@
                        formControlName="podNodeSelectorAdmissionPluginConfig"
                        fxFlex="100"></km-label-form>
         <div *ngIf="isPluginEnabled(admissionPlugin.EventRateLimit)">
-          <km-wizard-cluster-event-rate-limit [formControl]="form.get(Controls.EventRateLimitConfig)" [eventRateLimitConfig]="eventRateLimitConfig"></km-wizard-cluster-event-rate-limit>
+          <km-wizard-cluster-event-rate-limit [formControl]="form.get(Controls.EventRateLimitConfig)"
```

Comment:
This has been fixed in `master` so please revert this change and rebase your branch on latest `master`.